feature-libs/organization/administration/styles/_form.scss

```diff
@@ -5,6 +5,9 @@
       // align the height of the select with buttons
       max-height: 48px;
     }
+    @include media-breakpoint-down(sm) {
+      font-size: var(--cx-font-size, 12px);
```

Comment:
@Parthlakhani Sorry, I forgot to mention that at this time, we still use only SASS variables for handling font sizes, the smallest currently available is `type('7')` and is equal to 14px. Please run a search for `@include type('7');` in SCSS files to see how it should be implemented. If the size is not small enough then just set `white-space: normal` and let it flow in two rows, just in that field for `sm` mobile and B2B only. Once the select box is open the long options are visible in two rows. Keep in mind this will adjust with localization (client's phrasing and language) and with phone orientation, so there's a limit to what we can do here.